sylius/admin-order-creation-plugin/0.6/manifest.json

```diff
@@ -0,0 +1,12 @@
+{
+    "bundles": {
+        "Sylius\\AdminOrderCreationPlugin\\SyliusAdminOrderCreationPlugin": ["all"]
+    },
+    "copy-from-recipe": {
+        "config/": "%CONFIG_DIR%/"
+    },
+    "composer-scripts": {
+        "assets:install --symlink %PUBLIC_DIR%": "symfony-cmd",
+        "cache:clear": "symfony-cmd"
```

Comment:
These 2 lines should be removed as there are already part of framework bundle.

Reply:
Indeed, it has resulted in two `assets:install` commands.

Reply:
👍 Fixed 🚀